src/devices/csaferower/csaferower.cpp

```diff
@@ -90,16 +90,15 @@ void csaferower::onDistance(double distance) {
 
 }
 
-void csaferower::onStatus(uint16_t status) {
+void csaferower::onStatus(char status) {
     qDebug() << "Current Status received:" << status;
 }
 
 csaferowerThread::csaferowerThread() {}
 
 void csaferowerThread::run() {
     QSettings settings;
-    /*devicePort =
-        settings.value(QZSettings::computrainer_serialport, QZSettings::default_computrainer_serialport).toString();*/
+    deviceFilename = settings.value(QZSettings::csafe_rower, QZSettings::default_csafe_rower).toString();
```

Comment:
@marcelrv on android this is not needed. i guess you need to add a #ifdef here for linux only